src/plugins/console/public/application/models/legacy_core_editor/mode/worker/worker.js

```diff
@@ -2022,9 +2022,8 @@ ace.define(
       },
       // parses and returns the method
       method = function () {
-        const [first, ...rest] = text.split(' ');
-        text = first.toUpperCase() + rest.join(' ');
-        ch = ch.toUpperCase();
+        const re = /get|post|put|head|delete/gi
```

Comment:
what do you think to fix it in a not very pretty way?

example for `get`
![image](https://user-images.githubusercontent.com/20072247/145019805-d0e54959-4df9-41bf-9f23-beafa43011af.png)

I agree, maybe this is not very beautiful, but 100% safe

@mibragimov @sabarasaba  any ideas? But looks current fix also works